common/parallel/dirReader_default.go

```diff
@@ -0,0 +1,45 @@
+// +build !linux
+
+// TODO: this currently applies on Darwin. Should it? Or should we use the Linux implementation there?
```

Comment:
>TODO: this currently applies on Darwin. [](start = 3, length = 39)

I'm not sure, would you like me to check?

Reply:
Yes, if you could check, that would be great thanks.